lib/model/regular_polygon.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:ui';
+
+// TODO: Add comment.
+class RegularPolygon {
+  /// Create a new [RegularPolygon].
+  ///
+  /// The number of [sides] has to be supplied.
```

Comment:
This does not have to be mentioned in the doc comment, as it is apparent via the `required` keyword.